src/BookNavigator/search/search-provider.js

```diff
@@ -1,6 +1,6 @@
 import { html } from 'lit-element';
 import { nothing } from 'lit-html';
-
+import '@internetarchive/icon-search/icon-search';
```

Comment:
I can't comment on it, but there's an empty file just above this one: `provider-config.js`.

Reply:
TY